### PR TITLE
Stats counter

### LINKED ISSUE #6: [3pt]  - Add all items counter on the Homepage - Student A
Even if the API gives you the number of items, you will create a specific function to calculate it.

Make sure that the correct number is displayed on the Homepage.

Respect the following rules:

- Each counter should be implemented as a separate module.
- A counter function should look for specific DOM elements (e.g. for the comments counter it should look for comments) and make the counting based on what is actually displayed on the page.
- A counter function should cover all the edge cases you can think about.

### PR DESCRIPTION
**Pull request by rbhatt1999**
In this PR I do the following :

- Add all items counter on the Homepage

### CODE REVIEW

File: src/modules/Functions.js [hunk 1]
```diff
@@ -73,29 +83,41 @@ function DisplayPopup(data, examples) {
...
+function Counter(data) {
+//   const LikesCounter = document.querySelectorAll('.likes-counter');
+  ProductsCounter.innerHTML = data.products.length;
+  // data.likes.forEach(item => {
+  //     LikesCounter.forEach(counter=>{
+  //         if(item.item_id === counter.id) {
+  //             counter.innerHTML=item.likes
+  //         }
+  //     })
+  // })
```
Comment (codewanderlust) [inline src/modules/Functions.js]: Kindly remove unused code to make your work cleaner

**Review by codewanderlust — CHANGES_REQUESTED** [review]
###  STATUS: REQUIRED CHANGES ♻️

Hi @rbhatt1999  👋




Your project page is looking fantastic so far 🤩 !

However, there is still some room for improvement, which can help you have better code and make your project professional 😎

Highlights ✨

Linters are all passing ✔
Great work using the right GitHub Flow 💯
Commits are descriptive and meaningful 👍
Project Documentation (PR, README) is professional 👔
Congrats on your work so far! 👏

Required Changes ♻️
There are some issues that you still need to work on to go to the next project, but you are almost there!
For more information, check the comments under the review ⬇️

Optional suggestions ☑️
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 💯

Happy coding and keep it up! 🙌

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Nyame-Wolf in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/modules/Functions.js (hunk 1 above)
Comment (rbhatt1999) [reply src/modules/Functions.js]: Resolved

**Review by codewanderlust — APPROVED** [review]
###   Approved ✔

Hi @rbhatt1999  ,

Great job, Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉



(Highlights) Good Points: 👍

You documented your work professionally. ✔
GitHub flow is used correctly. ✔
All linters checks are ok. ✔

(Changes Required) Aspects to improve: ♻️
N/A

[Optional] suggestions:
N/A

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**